charts/pulsar/values.yaml

```diff
@@ -1105,6 +1105,11 @@ proxy:
     ## Restrict traffic through the load balancer to specified IPs on providers supporting it.
     # loadBalancerSourceRanges:
     #   - 10.0.0.0/8
+    nodePorts:
```

Comment:
Please add documentation for `nodePorts` as a comment

Reply:
I guess it would be also necessary to document that the service type must be set to `NodePort` for this to be effective. ([comment on another issue](https://github.com/apache/pulsar-helm-chart/issues/497#issuecomment-2118784954))

Reply:
absolutely forgot to add that!

Addressed by 90c06e8

Reply:
Thank you very much for the review!

Reply:
Just validated on my local k3d, we should be good!